### ISSUE TITLE
Missing "Accept" Request Header in generated files based on OAS 3.0 

### ISSUE THREAD

**Issue by guillaumeserale** [christianhelle/refitter#107]
When Refitter generates Refit interfaces from a OAS 3.0 spec file, it doesn't add the ["Accept" header ](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Accept), which necesssary for ASP.NET Core's [ProblemDetails Middleware](https://learn.microsoft.com/en-us/aspnet/core/fundamentals/minimal-apis/handle-errrors?view=aspnetcore-7.0#problem-details) to function properly.

According to [OAS 3.0 specification](https://swagger.io/docs/specification/describing-responses/), The `content` property in `responses` contains the MIME type expected in the response.

I'm suggesting to add an optional parameter `addAcceptHeaders` to add the accept header.
Examples

- without  `addAcceptHeaders`:
```csharp/// <summary>
        /// Multiple status values can be provided with comma separated strings
        /// </summary>
        [Get("/pet/findByStatus")]
        Task<ICollection<Pet>> FindPetsByStatus([Query(CollectionFormat.Multi)] IEnumerable<Anonymous> status);
```

- with `addAcceptHeaders`:
```csharp/// <summary>
        /// Multiple status values can be provided with comma separated strings
        /// </summary>
        [Headers("Accept: application/json")]
        [Get("/pet/findByStatus")]
        Task<ICollection<Pet>> FindPetsByStatus([Query(CollectionFormat.Multi)] IEnumerable<Anonymous> status);```

**Comment by christianhelle** [comment]
@guillaumeserale thanks for reporting this

**Comment by christianhelle** [comment]
@all-contributors please add @guillaumeserale for bug

**Comment by christianhelle** [comment]
Implemented in #108
